### ISSUE TITLE
remove (most) submodules

### ISSUE THREAD

**Issue by sdumetz** [Holusion/eCorpus#64]
See [this script](https://stackoverflow.com/questions/28215244/convert-git-submodule-to-subtree)

Effort ongoing [here](https://github.com/Smithsonian/dpo-voyager/compare/master...Holusion:dpo-voyager:subtree)

Squashing (or not) the commits from the submodules into the same folder is easy enough. Removing dead code might be more work intensive.

**Comment by sdumetz** [comment]
Identified a bunch of unused files with this [knip](https://knip.dev/) config:
```
"knip": {
        "entry": [
            "source/client/ui/explorer/MainView.ts",
            "source/client/ui/story/MainView.ts",
            "source/client/ui/launcher/MainView.ts",
            "source/client/ui/mini/MainView.ts"
        ],
        "project": ["source/client/**/*.ts", "libs/**/*.ts"],
        "paths":{
            "client/*": [ "./source/client/*" ],
            "@ff/core/*": [ "./libs/ff-core/source/*" ],
            "@ff/graph/*": [ "./libs/ff-graph/source/*" ],
            "@ff/browser/*": [ "./libs/ff-browser/source/*" ],
            "@ff/ui/*": [ "./libs/ff-ui/source/*" ],
            "@ff/react/*": [ "./libs/ff-react/source/*" ],
            "@ff/three/*": [ "./libs/ff-three/source/*" ],
            "@ff/scene/*": [ "./libs/ff-scene/source/*" ]
        }
    }
```

**Comment by sdumetz** [comment]
See https://github.com/Smithsonian/dpo-voyager/pull/284

**Comment by sdumetz** [comment]
Merged in Voyager v0.43.0
